technical-reports/resolver/rationale.md

```diff
@@ -1,48 +0,0 @@
-## Rationale
-
-### Context and Problem Statement
-
-In design systems, especially those at scale, managing multiple themes, modes, brands, and other modifiers can become complex. Designers and developers often struggle with:
-
-- **Naming Conflicts**: Using complex naming conventions to differentiate tokens across different themes or modes, leading to brittle and hard-to-maintain token names.
-- **Scalability Issues**: Difficulty in scaling token management when new dimensions are added, such as additional themes or brands.
-- **Tooling Limitations**: Existing tools may not support multi-dimensional token resolution, forcing teams to create workarounds that complicate their workflows.
-- **Separation of Concerns**: Embedding resolution logic within token files mixes data with behavior, violating separation of
-  concerns.
-
-### Parameters
-
-- **Simplicity**: The solution should keep token files simple and editable by hand, without requiring complex tooling.
-- **Scalability**: It should support arbitrary dimensions (e.g., themes, modes, brands) without significant overhead.
-- **Modularity**: Tokens should be organized in a way that allows for modularity and reusability.
-- **Separation of Concerns**: Resolution logic should be externalized from token definitions.
-
-### Background
-
-Existing approaches, such as embedding modifiers directly into token names or using global theme files, have limitations. They often lead to:
-
-- **Global Namespace Pollution**: All tokens exist in a single global namespace, increasing the likelihood of naming conflicts.
-- **High Cognitive Load**: Designers and developers must remember complex naming conventions and how different tokens relate across dimensions.
-- **Performance Issues**: Resolving tokens in systems with many dimensions can be slow due to the combinatorial explosion of permutations.
-
-### Motivation
-
-The Resolver Specification aims to:
```

Comment:
A specification shouldn’t “aim to” accomplish anything. “Do or do not, there is no try.” etc. A specification should describe what it means to describe, and nothing more.

Reply:
All the following bullet points are all unvalidated expectations of using this specification. However, I’d vote to remove it because they are unverifiable. We can’t measure any improvements without first outlining, at length, the original statuses of all cases. But also, doing so is beyond the purpose of any specification, which again, only describes syntax and format.